pkg/scheduler/profile/profile.go

```diff
@@ -95,9 +95,6 @@ func (v *cfgValidator) validate(cfg config.KubeSchedulerProfile, f framework.Fra
 	if v.m[f.ProfileName()] != nil {
 		return fmt.Errorf("duplicate profile with scheduler name %q", f.ProfileName())
 	}
-	if len(f.ListPlugins().QueueSort.Enabled) != 1 {
```

Comment:
Now you can get rid of `f` as parameter :)

Reply:
The `validate` call has to come after framework instantiation so that we can check for expanded QueueSort plugins across all profiles.

Also for that reason, we still need the framework to be passed to this validate function, because the expansion doesn't actually modify the Profile that is passed to `NewFramework`. MultiPoint plugins are only expanded into the framework's internal representation of each extension.

Reply:
Oh, we should have moved that check to `NewFramework` as well.

Reply:
`NewFramework` only handles one profile at a time, this check is making sure that there are no conflicting QueueSort plugins across all instantiated profiles.

Reply:
Ahhh, I forgot about that.

LGTM then. @ahg-g anything else before squashing?